### ISSUE TITLE
Node 18 support

### ISSUE THREAD

**Issue by msotnikov** [expo/expo-cli#4478]
LTS version 18 released in april. https://nodejs.org/ko/blog/announcements/v18-release-announce/

### Related issues
https://github.com/expo/expo-cli/issues/4457
https://github.com/expo/expo-cli/issues/4426

### Do you understand that any feature requests opened in the core Expo repository will be closed?

Yes

**Comment by brentvatne** [comment]
last week we released expo sdk 46 beta which includes our new cli tool that supports node 18, among many other improvements. https://blog.expo.dev/expo-sdk-46-beta-is-now-available-9dfee4040aa7

we'll do the full release of this soon. we don't plan on adding support to the old expo cli (the globally installed package found in this repository).

**Comment by kbrandwijk** [comment]
> LTS version 18 released in april. https://nodejs.org/ko/blog/announcements/v18-release-announce/

Not that it matters much with support added in the new CLI in SDK 46, but the _initial_ release of Node 18 was in April. LTS actually starts October 25th, giving tool developers (like us) about 6 months to make sure everything works _before_ it becomes LTS. More information here: [https://nodejs.org/en/about/releases/](https://nodejs.org/en/about/releases/).

**Comment by msotnikov** [comment]
Thank you for you answer!

**Comment by orome** [comment]
I'm late to the party so forgive the dumb question. 

Does this mean that I can replace 16 (which I'm currently using) with 18 for use with my SDK 46  ([Expo 46 => expo-deve-client 1.2.X => react-native 0.69.X][1]) apps?

Can I just I upgrade to 18, e.g., using
```
brew uinistall node@16
brew install node@18
```

  [1]: https://docs.expo.dev/development/compatibility/

**Comment by codeflyer** [comment]
> last week we released expo sdk 46 beta which includes our new cli tool that supports node 18, among many other improvements. https://blog.expo.dev/expo-sdk-46-beta-is-now-available-9dfee4040aa7
> 
> we'll do the full release of this soon. we don't plan on adding support to the old expo cli (the globally installed package found in this repository).

I haven't found any mention in the Change.log related to node 18.
Checking the package.json of the `expo-cli` package the engine are still limited to the version 16

```
  "engines": {
    "node": ">=12 <=16"
  },
```

Even if with node 18 it works locally using it breaks the `expo-github-action`.

>  error expo-cli@6.0.8: The engine "node" is incompatible with this module. Expected version ">=12 <=16". Got "18.12.0"
>   error Found incompatible module.

There are some news about the full support?

**Comment by orome** [comment]
I'm confused: `expo doctor` gives
```
WARNING: expo-cli has not yet been tested against Node.js v18.12.1.
If you encounter any issues, please report them to https://github.com/expo/expo-cli/issues

expo-cli supports following Node.js versions:
* >=12.13.0 <15.0.0 (Maintenance LTS)
* >=16.0.0 <17.0.0 (Active LTS)
```
but the LTS specification matching current releases [should be](https://github.com/nodejs/Release#nodejs-release-working-group)
```
* >=18.0.0 <19.0.0 (Active LTS)
```
right?
